docs/clients/promtail/configuration.md

```diff
@@ -979,43 +1017,47 @@ sync_period: "10s"
 
 ## Example Docker Config
 
+It's fairly difficult to tail docker files on a standalone machine because they are in different locations for every OS.  We recommend the [docker logging driver](../../../cmd/docker-driver/README.md) for local docker installs or docker-compose.
+
+If running in a kubernetes environment, you should look at the defined configs which are in [helm](../../../production/helm/promtail/templates/configmap.yaml) and [jsonnet](../../../production/ksonnet/promtail/scrape_config.libsonnet), these leverage the prometheus service discovery libraries (and give promtail it's name) for automatically finding and tailing pods.  The jsonnet config explains with comments what each section is for.
+
+
+## Example Static Config
+
+While promtail may have been named for the prometheus service discovery code, that same code works very well for tailing logs without containers or container environments directly on virtual machines or bare metal.
+
 ```yaml
 server:
   http_listen_port: 9080
   grpc_listen_port: 0
 
 positions:
-  filename: /tmp/positions.yaml
+  filename: /var/log/positions.yaml ①
 
 client:
   url: http://ip_or_hostname_where_Loki_run:3100/loki/api/v1/push
 
 scrape_configs:
  - job_name: system
    pipeline_stages:
-   - docker:
    static_configs:
    - targets:
       - localhost
      labels:
-      job: varlogs
-      host: yourhost
-      __path__: /var/log/*.log
-
- - job_name: someone_service
-   pipeline_stages:
-   - docker:
-   static_configs:
-   - targets:
-      - localhost
-     labels:
-      job: someone_service
-      host: yourhost
-      __path__: /srv/log/someone_service/*.log
+      job: varlogs  ②
+      host: yourhost ③  
+      __path__: /var/log/*.log  ④
 ```
+① This location needs to be writeable by promtail.  
+② A `job` label is fairly standard in prometheus and useful for linking metrics and logs.   
+③ A `host` label will help identify logs from this machine vs others.  
+④ The path matching uses [@bmatcuk's glob library](https://github.com/bmatcuk/doublestar)  
```

Comment:
I've been trying to move away from using these because you can't copy and paste it and a little strange to see (at least for me). Can you make these inline YAML comments instead?